### DISCUSSION TITLE
rime导入拆字词库后在输入拼音中包含u的词语的时候会卡一下才弹出候选词列表

### DISCUSSION THREAD

**Discussion by francis707** [rime/home#764] [Q&A]
比如打`所以`的时候，`suoyi`输入到`y`的时候，会卡一下才弹出候选列表，还有`属于`、`做事`、`输入`等等一些词语也有这种现象

**Comment by francis707** [comment]
找到原因了，原来是我引入的一个2万7千多词条的拆字词库：[luna_pinyin_simp.chaizi.dict.zip](https://github.com/rime/home/files/6230400/luna_pinyin_simp.chaizi.dict.zip)
比如（设置了`u`为触发字母）：
```
淼	ushuishuishui	1
焱	uhuohuohuo	1
```
结果在输入拼音中包含`u`的词语时，即使不是以`u`开头也会触发拆字词库，导致卡顿，把这个词库注释掉就好了
这个问题怎么解决呢？是否有什么设置可以避开这个问题？

**Comment by lotem** [comment]
雖然編碼裏面包含拼音，但嚴格來說這種編碼不是拼音。因此這些數據不能混入拼音詞典。
請另建一個詞典，並使用獨立於拼音的`table_translator`查詢。或刪去前綴`u`之後與原裝方案的五筆畫反查詞典合併，將`u`設置爲筆畫、拆字反查前導鍵。

**Reply by francis707** [reply]
好的，谢谢，我回头看看 `table_translator` 和 `反查` 相关的文档
